nx-X11/lib/X11/Xlibint.h

```diff
@@ -880,10 +953,6 @@ extern void _XEatData(
     Display*		/* dpy */,
     unsigned long	/* n */
 );
-extern void _XEatDataWords(
```

Comment:
Yes, all that has been fixed in the following commit. I will try to move these changes to to the right commit